content/influxdb/v2.0/install.md

```diff
@@ -545,6 +546,25 @@ Always inspect YAML manifests before running `kubectl apply -f <url>`!
 
 {{% /tab-content %}}
 <!--------------------------------- END kubernetes ---------------------------->
+<!--------------------------------- BEGIN Rasberry Pi ------------------------->
+{{% tab-content %}}
+
+To run InfluxDB on Raspberry Pi, you will need:
+
+- a Raspberry Pi 4+ or 400
+- a 64-bit operating system.
+  We recommend installing a [64-bit version of Ubuntu](https://ubuntu.com/download/raspberry-pi)
+  of Ubuntu Desktop or Ubuntu Server compatible with 64-bit Raspberry Pi.
+
+Follow the [Linux installation instructions](/influxdb/v2.0/install/?t=Linux) to install InfluxDB on a Raspberry Pi.
+
+#### Support for 32-bit systems
+
+If you have a 32-bit Raspberry Pi, [use Telegraf](/{{< latest "telegraf" >}}/)
+to collect and send data to [InfluxDB Cloud](/influxdb/cloud).
```

Comment:
You can also send the Data to InfluxDB OSS... not sure how you want to surface that.  

Perhaps we need to point folks at the Free Tier account setup for hobbyists?  While I don't want our docs to turn into cross selling/marketing.  I feel like describing the options here isn't out of bounds....
Send data to: 
InfluxDB Cloud -- Free Tier -- within the various established limits
InfluxDB Cloud -- PAYG -- relaxed restrictions (per account management page)
InfluxDB OSS -- run it yourself, but...on a separate machine.....